test/studies/bale/indexgather/ig-variants.chpl

```diff
@@ -56,16 +56,28 @@ proc testit(mode: Mode, param explicit, printStats) {
   var t: Timer; t.start();
   select mode {
     when Mode.directIndexLocal {
-      forall i in D2 do
-        copy(tmp.localAccess[i], A[rindex.localAccess[i]], explicit);
+      forall i in D2 {
+        if explicit then
+          unorderedCopy(tmp.localAccess[i], A[rindex.localAccess[i]]);
+        else
+          tmp.localAccess[i] = A[rindex.localAccess[i]];
+      }
     }
     when Mode.directIndex {
-      forall i in D2 do
-        copy(tmp[i], A[rindex[i]], explicit);
+      forall i in D2 {
+        if explicit then
+          unorderedCopy(tmp[i], A[rindex[i]]);
+        else
+          tmp[i] = A[rindex[i]];
+      }
     }
     when Mode.zipArray {
-      forall (t, r) in zip(tmp, rindex) do
-        copy(t, A[r], explicit);
+      forall (t, r) in zip(tmp, rindex) {
+        if explicit then
```

Comment:
Mind writing these as one-liners to match the promotion case? i.e. 

```chpl
if explicit then unorderedCopy(t, A[r]);
            else t = A[r];
```

Reply:
If I do that, the first one does not fit on one line.
I could do it everywhere else, let the line wrap, or change the promoted case to be multi-line.

Reply:
I don't mind lines being over 80 chars, but if you want to keep it under let's change the promoted case to be multi-line.

Reply:
OK. I made them multi-line.